.circleci/config.yml

```diff
@@ -50,7 +50,13 @@ jobs:
             - run:
                   name: Unit test Lightning Web Components
                   command: |
-                      npm run test:unit -- -- --runInBand
+                      npm run test:unit:coverage -- -- --runInBand
+            - persist_to_workspace:
+                  root: ~/
+                  paths:
+                      - .sfdx/*
+                      - project/*
+                      - coverage/*
```

Comment:
Can we add a comment here about what this does? (i.e. we're storing code coverage results)

Reply:
Added